.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "dropwizard-swagger"]
```

Comment:
Why did you add this as a submodule? The dependency declaration in the pom.xml should suffice